oktapam/data_source_cloud_connection_test.go

```diff
@@ -0,0 +1,63 @@
+package oktapam
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/okta/terraform-provider-oktapam/oktapam/constants/attributes"
+)
+
+func createTestAccDataSourceCloudConnectionInitConfig(identifier string) string {
+	const configFormat = `
+	resource "oktapam_cloud_connection" "test-cloud-connection-1" {
+		name = "%s-1"
+		provider = "aws"
```

Comment:
In the attributes file, you have provided the name of this attribute as `cloud_connection_provider`.